account_invoice_view_payment/views/account_payment_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_payment_invoice_form" model="ir.ui.view">
+        <field name="name">account.payment.invoice.form</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id"
+               ref="account.view_account_payment_invoice_form"/>
+        <field name="arch" type="xml">
+            <button name="post" position="after">
+                <button name="post_and_open_payment" class="oe_highlight"
+                        string="Validate &amp; View Payment" type="object"/>
```

Comment:
@mreficent Don't you think you also need to add the same group `groups="account.group_account_user"/>` here ?!